### ISSUE TITLE
System unavailable: test-osuosl-aix72-ppc64-4 out of disk space

### ISSUE THREAD

**Issue by Haroon-Khel** [adoptium/infrastructure#3029]
https://ci.adoptium.net/computer/test-osuosl-aix72-ppc64-4/ is low on disk space and it is down in Jenkins
```
Disk space is too low. Only 6.849GB left on /home/jenkins.
```
The following directories are taking up a bit of space
```
root@adopt10:[/home/jenkins/workspace/jdk-aix-dev-build/JDK_VERSION]du -sh *
332M  jdk
5.0G  jdk11u
4.3G  jdk17u
16M   jdk18u
4.0G  jdk8u
```

**Comment by sxa** [comment]
NOTE: The agent mentioned in the title has no labels therefore nothing is running through it.
The machine is, however, running two jenkins agents. The other one is for [build-osuosl-aix72-ppc64-1](https://ci.adoptium.net/manage/computer/build%2Dosuosl%2Daix72%2Dppc64%2D1/) and that's the one that's chewing through space.

**Comment by sxa** [comment]
@Haroon-Khel Since there is no available disk available for expansion I suggest that for now, during the release cycle, we perform the following tasks on [all machines labelled build72](https://ci.adoptium.net/label/build72/).
- Disable the [jdk-dev-aix-build](https://ci.adoptium.net/job/jdk-aix-dev-build/) job in jenkins to stop it running
- Remove jdk-aix-dev-build from the workspace directory on each of the labelled machines

We can then look at future options once we're out of the release cycle. I will note that the AIX 7.1 build boxes have 50% more space available to the workspaces, so ideally we'd have a comparable amount (ideally larger) on the AIX 7.2 systems we will use for build going forward.

@aixtools Are we still quite restricted in terms of disk space available on the OSUOSL AIX machins?

**Comment by Haroon-Khel** [comment]
i've disabled the jdk-dev-aix-build and removed its directory on each of the build72 machines. Theres >20G free on each of the machines now

**Comment by aixtools** [comment]
I don´t have the sizes of systems memorized, but I'll look for some space.

There are a few changes that need to be made with VIOS (which requires taking a frame down). Once that is done I could move some of the adopt systems where there is storage available.

**Comment by aixtools** [comment]
See #3052 and #3053 
test-osuosl-aix72-ppc64-4 is _actually_ build-osuosl-aix72-ppc64-1

Rather amazing this was not a problem much earlier :)

imho: can be closed in parallel with #3052 and #3053
